nilearn/interfaces/fmriprep/load_confounds_scrub.py

```diff
@@ -87,6 +89,15 @@ def extract_outlier_regressors(confounds):
         outliers = pd.DataFrame()
     confounds = confounds.loc[:, confounds_cols]
     sample_mask = _outlier_to_sample_mask(outliers)
+
+    if sample_mask is not None and sample_mask.size == 0:
+        warnings.warn(
+            category=RuntimeWarning,
+            message="All time points in the confounds were marked as "
+            "outliers. This would lead to all volumes in the time "
+            "series to be scrubbed.",
```

Comment:
I would keep only volume or time points, rather than using them interchangeably.
Also we are technically marking the volume as the motion outliers, not the confounds.

```suggestion
            message="All volumes were marked as motion outliers. "
            "This would lead to all volumes in the time "
            "series to be scrubbed.",

```